### ISSUE TITLE
This is awesome!

### ISSUE THREAD

**Issue by kyle-ssg** [TheMagoo73/flagsmith-react#7]
Hey, thought I'd reach out and say this project looks great - love the hook based approach and can definitely see this being used. Hell, I'd use it!

There are a couple small things that come to mind that I think could be really helpful in setting out a strong foundation for this:

- It would be amazing if the Flagsmith provider could have an optional flagsmith implementation passed to it, with very few changes this will make this automatically work for react-native. Here's what I'd maybe think for this :

```
import flagsmith from 'react-native-flagsmith'
<FlagsmithProvider
        flagsmith={flagsmith}
        environmentId="YOUR_FLAGSMITH_ENVIRONMENT_ID"
 />
```

- The FlagsmithContext could quite easily be typed by importing and reusing https://github.com/Flagsmith/flagsmith-js-client/blob/master/flagsmith/index.d.ts

Overall though great stuff excited to try it out.

**Comment by TheMagoo73** [comment]
Thanks for the kind words, and awesome suggestions! In the backlog for v1 👍

**Comment by TheMagoo73** [comment]
Closed by #9
